<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стеганография</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2E3440;
color: #D8DEE9; </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
    border: 1px solid #4C566A;
    background-color: #2E3440;
    border-radius: 5px;
}

QTabBar::tab {
    background-color: #3B4252;
    color: #D8DEE9;
    padding: 8px 16px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    border: 1px solid #4C566A;
    border-bottom: none;
    margin-right: 2px;
}

QTabBar::tab:selected {
    background-color: #4C566A;
    color: #ECEFF4;
}

QTabBar::tab:hover:!selected {
    background-color: #434C5E;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSteganography">
       <attribute name="title">
        <string>Стеганография</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBoxEmbed">
              <property name="title">
               <string>Внедрение сообщения</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox {
    border: 1px solid #4C566A;
    border-radius: 5px;
    margin-top: 1ex;
    font-weight: bold;
    color: #88C0D0;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLineEdit" name="editEmbedMessage">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
    background-color: #3B4252;
    color: #D8DEE9;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 4px;
}</string>
                 </property>
                 <property name="placeholderText">
                  <string>Сообщение для внедрения</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSelectImageEmbed">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #5E81AC;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
                 </property>
                 <property name="text">
                  <string>Выбрать изображение</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnEmbed">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #A3BE8C;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #B9D39C;
}</string>
                 </property>
                 <property name="text">
                  <string>Внедрить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSaveImage">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #5E81AC;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
                 </property>
                 <property name="text">
                  <string>Сохранить изображение</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxExtract">
              <property name="title">
               <string>Извлечение сообщения</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox {
    border: 1px solid #4C566A;
    border-radius: 5px;
    margin-top: 1ex;
    font-weight: bold;
    color: #88C0D0;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="btnSelectImageExtract">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #5E81AC;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
                 </property>
                 <property name="text">
                  <string>Выбрать изображение</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnExtract">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #A3BE8C;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #B9D39C;
}</string>
                 </property>
                 <property name="text">
                  <string>Извлечь</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="labelImage">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    background-color: #3B4252;
    border: 1px solid #4C566A;
    border-radius: 5px;
}</string>
              </property>
              <property name="text">
               <string>Изображение будет отображаться здесь</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textOutput">
              <property name="styleSheet">
               <string notr="true">QTextEdit {
    background-color: #3B4252;
    color: #D8DEE9;
    border: 1px solid #4C566A;
    border-radius: 5px;
    padding: 5px;
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Результаты будут отображаться здесь</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEmpty">
       <attribute name="title">
        <string>Пустая вкладка</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelEmpty">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #D8DEE9;
    font-size: 16pt;
    font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>Пустая вкладка для будущего расширения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

